src/it/scala/org/ergoplatform/it/Docker.scala

```diff
@@ -296,5 +299,29 @@ object Docker {
     m.get(s"$containerPort/tcp").get(0).hostPort().toInt
 
   val DefaultConfigTemplate: Config = ConfigFactory.parseResources("template.conf")
-  val NodeConfigs: Config = ConfigFactory.parseResources("nodes.conf")
+  val NodeConfigs: Config = ConfigFactory.parseResources("nodes.conf").resolve()
+
+  val networkNamePrefix: String = "ergo-itest-"
+
+  def cleanupResources(): Unit = {
+    log.debug("Cleaning up Docker resources")
+    val client = DefaultDockerClient.fromEnv().build()
+    // remove containers
+    client.listContainers(ListContainersParam.allContainers()).asScala
+      .filter(_.names.asScala.head.startsWith("/" + networkNamePrefix))
+      .foreach(c => client.removeContainer(c.id, RemoveContainerParam.forceKill))
+    // removes networks
+    client.listNetworks(ListNetworksParam.customNetworks).asScala
+      .filter(_.name().startsWith(networkNamePrefix))
+      .foreach(n => client.removeNetwork(n.id))
+  }
+
+  def cleanupResourcesIfNeeded(): Unit =
+    Try { Docker.NodeConfigs.getBoolean("testing.integration.cleanupDocker") }
```

Comment:
It's better to use `ficus` here as we do at the project:
```
import net.ceedubs.ficus.Ficus._
```
and then:

```
val shouldCleanup = Docker.NodeConfigs.getOrElse[Boolean]("testing.integration.cleanupDocker", false)
```

Reply:
Nice catch. I missed that. Thanks.